spec/abstractops.html

```diff
@@ -944,9 +952,9 @@ <h1>ISO 8601 grammar</h1>
           TimeZoneIANANameTail
 
       TimeZoneBracketedName :
-          TimeZoneIANAName
-          `Etc/GMT` ASCIISign Hour
           TimeZoneUTCOffsetName
+          `Etc/GMT` ASCIISign UnpaddedHour
+          TimeZoneIANANameTail but not `Etc/GMT` ASCIISign UnpaddedHour
```

Comment:
I'm concerned about the "Etc/GMT±\<hour>" text being excluded from |TemporalTimeZoneIdentifier|, which includes |TimeZoneIANAName| but not |TimeZoneBracketedName|. It would be better to move ``` `Etc/GMT` ASCIISign UnpaddedHour``` to the former, as suggested in #1993:
```
      TimeZoneIANAName :
          `Etc/GMT` ASCIISign UnpaddedHour
          TimeZoneIANANameTail but not `Etc/GMT` ASCIISign UnpaddedHour

      TimeZoneBracketedName :
          TimeZoneIANAName
          TimeZoneUTCOffsetName
```